nordicsemi/__main__.py

```diff
@@ -834,6 +838,12 @@ def convert_version_string_to_int(s):
 
 def thread(package, port, address, server_port, panid, channel, jlink_snr, flash_connectivity,
            sim, rate, reset_suppress):
+    """
+    Perform a Device Firmware Update on a device with a bootloader that supports Thread DFU.
```

Comment:
`Perform a Device Firmware Update on a device that supports Thread DFU.`

Thread DFU is performed in the background while application is running, not in the bootloader, hence I would not mention it here.